### ISSUE TITLE
Add templates for asset-functions that serve Javascript and CSS

### ISSUE THREAD

**Issue by parruda** [FaaStRuby/faastruby-cli#6]
The idea is to create a template for the command `faastruby new` that would initialize a folder with a function already prepared to serve Javascript and CSS.
The command could be, for example:
```ruby
~$ faastruby new FUNCTION_NAME --template assets
```
Note that we must have templates for all the supported runtimes (--runtime).

The resulting folder could look like this:
```
function_folder
├── assets
│   ├── javascript
│   │   └── application.js
│   └── stylesheets
│       └── application.css
├── spec
│   ├── handler_spec.rb
│   └── spec_helper.rb
├── Gemfile
└── handler.rb
```

Example requests:
* `/workspace-name/function-name?css=application.css`
* `/workspace-name/function-name?js=application.js`

And an example Ruby function could be like:
```ruby
def handler(event)
  return render(css: File.read("assets/stylesheets/#{event.query_params['css']}")) if event.query_params.has_key?('css')
  return render(js: File.read("assets/javascript/#{event.query_params['js']}")) if event.query_params.has_key?('js')
end
```
There's also the need for sanitizing the input. For example, we don't want to allow `js=../foo.js` (which would not result in anything bad because the platform is secured against stuff like that - but it is better to avoid anyways). So there's a need for some sort of regular expression to remove everything from`css=` and `js=` but letters, numbers, dashes, underlines and one period.
The above is just a suggestion. Feel free to propose something better.

**Comment by parruda** [comment]
It would also be great if this could generate from a template that is hosted in a remote Git repository. For example:
```bash
~$ faastruby new FUNCTION_NAME --template github:parruda/faastruby-assets-template
```

**Comment by parruda** [comment]
Closed in favour of #8
